libs/community/langchain_community/embeddings/huggingface.py

```diff
@@ -327,7 +329,7 @@ def _default_api_url(self) -> str:
 
     @property
     def _headers(self) -> dict:
-        return {"Authorization": f"Bearer {self.api_key.get_secret_value()}"}
+        return {"Authorization": f"Bearer {self.api_key.get_secret_value()}"}|self.additional_headers
```

Comment:
The merge operator was introduced in Python 3.9 for dictionaries, thus this won't work for python 3.8.2 (minimum version supported by langchain). Considering using:

```suggestion
        return {"Authorization": f"Bearer {self.api_key.get_secret_value()}", **self.additional_headers}
```